### ISSUE TITLE
Unable to change Lottie Animation 

### ISSUE THREAD

**Issue by serenamoonx** [FlutterFlow/flutterflow-issues#3779]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When I switch the lottie animation to a different asset, it doesn't change and it reverts to the previous asset. This is only in  desktop. In order to switch it, I have to go into the web version to switch it and then it will pop up in the local run.

### Expected Behavior

I should be able to switch out my lottie animations at will.

### Steps to Reproduce

1. Go to an old page with a lottie animation asset already chosen
2. In the dropdown for Asset Animation, switch the asset

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4gzcmAx5dgrsdc1br+cO9VhmE7JVh+U4c0u9tuGEgdCLbyD4lzf8/BQBJtXbyndkFEM06guDk1+aiKm9z1MvE4PgSubtVByMx2agLjRly5MrK2CJCoPGtuGshUfRGu07SzgAomPvVfWXA9216UPtitd3bXGLL6OUUNO/yKGJjXmAfuQ0SLb3kNh1JWeC/v

### Visual documentation

[https://github.com/user-attachments/assets/adfc5635-5030-4960-8a47-14cb9ebfd4c0](https://github.com/user-attachments/assets/adfc5635-5030-4960-8a47-14cb9ebfd4c0)

### Environment

```markdown
- FlutterFlow version: 4.1.88 (released sept 6, 2024)
- Platform: MacOs Desktop Local Run
- Browser name and version: N/A
- Operating system and version affected: macOs Monterey
```

### Additional Information

It slows me down a lot since I have to keep switching between local and web versions in order to get the project down

**Comment by rzambroni** [comment]
Hello @serenamoonx,

Thank you for your report! I wasn’t able to replicate the issue you mentioned—it seems to be working fine on my end. I also noticed that you were able to switch the animation in your project, and I was able to do the same without any issues. Could you let me know if you're still experiencing the problem?

Looking forward to your response!

**Comment by serenamoonx** [comment]
@rzambroni Yes, I'm still having the problem. I can only change it in the web version. The problem is only with the local run version

**Comment by rzambroni** [comment]
Thank you for providing those additional details. I was able to replicate the issue you described in the macOS app, and I've forwarded it to our engineering team for further investigation. We appreciate your report and will keep you updated as soon as we have more information!

**Comment by AlexP-cyber** [comment]
same for me - lottie animation can **NOT** be chosen on **BOTH** desktop and web version on Windows.... ff 4.1.88
Blocks me a bit, if it's possible please up its priority

**Comment by SebyOnGitHub** [comment]
Same problem here.

**Comment by serenamoonx** [comment]
How is this issue even worse now? After the latest update, I can't change the lottie animation on the local run or the web version. This is a big problem

**Comment by AlexP-cyber** [comment]
workaround : you need to remove lottie widget and add it again with new animation.

**Comment by nunezal** [comment]
Same issue here

**Comment by walik-walik** [comment]
Same Issue here, Lottie Anmiation Crashes with Error: Another exception was thrown: PersistedOffset: is in an unexpected state.

**Comment by dominikmucklow** [comment]
Same issue. Cannot change the Lottie animation. Workaround is to delete it and create a new one. Any newly added animations are also listed "This asset is not directly used in this project" in the Media Assets tab. Both on MacOS and Web versions.
